### DISCUSSION TITLE
2023년 11월 04일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#93] [General]
# 2023년 11월 04일

TODO 실행 계획 정리 [startTime:: 15:00]  [endTime:: 16:00] 면접 키워드 작성하기 [startTime:: 16:00] [endTime:: 18:00] LOG 배운 것 느낀 것 인간의 시간은 원형으로 돌지 않고 직선으로 나아간다.
행복은 반복의 욕구이기에 인간이 행복할 수 없는 것도 이런 이유 때문이다. 배신한다는 것은 줄…

https://vero.wiki/2023-11-04/

**Comment by carsago** [comment]
쿤데라 좋아요

**Reply by Cyma-s** [reply]
처음 읽어봤는데 좋더라고요
